styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -317,12 +319,15 @@ public void create(Environment environment) {
     closer.register(publisher);
 
     final ScheduledExecutorService executor = executorFactory.create(3, schedulerTf);
-    closer.register(executorCloser("scheduler", executor));
+    closer.register(executorCloser("scheduled-executor", executor));
```

Comment:
Putting on my uninitiated reader hat again: now we have both executor and schedulerExecutor in Scheduler, pretty confusing to read. And no clue which one of those is controlled by "styx.scheduler-threads" configuration parameter. And not sure why one of them uses newWorkStealingPool and the other uses some different kind of factory. (Would be great to add meaningful distinctions between the names, and meaningful connection with the configuration parameter name.)